src/method/method_invoke_group.cpp

```diff
@@ -210,7 +228,7 @@ namespace cubmethod
 	  }
```

Comment:
not necessary to check m_connection is null?

Reply:
If m_connection is nullptr, get_socket() will returns INVALID_SOCKET. and the method_invoke's will handle this case as connection failed error.